index.html

```diff
@@ -0,0 +1,181 @@
+              <div class="col-7 col-md-12 px-md-0 d-flex justify-content-center align-content-center align-items-center"><p class="text-white font-1px font-sm-18px font-md-20px text-md-center">Share your thoughts and opinions with our recognized software developers for each topic.</p></div>
+            </div>
+          </div>
+          <div class="program-block container-md program-cont-md p-20 my-2 mx-md-2 overflow-hidden">
+            <div class="row flex-nowrap px-3 d-md-flex flex-md-column align-items-md-center gap-md-1">
+              <div class="col-1 col-sm-2 col-md-12 d-flex justify-content-center align-content-center align-items-center"><img class="program-logo" src="./images/logos/workshop.png" alt="workshop-logo"></div>
+              <div class="col-4 col-sm-3 px-md-0 col-md-12 d-flex justify-content-start justify-content-md-center align-content-center align-items-center"><h3 class="color-red font-lato fw-bolder font-20px font-sm-25px">Workshop</h3></div>
+              <div class="col-7 col-md-12 px-md-0 d-flex justify-content-center align-content-center align-items-center"><p class="text-white font-1px font-sm-18px font-md-20px text-md-center">Create your own work using open source licensed frameworks rather than just be an spectator.</p></div>
+            </div>
+          </div>
+          <div class="program-block container-md program-cont-md p-20 my-2 mx-md-2 overflow-hidden">
+            <div class="row flex-nowrap px-3 d-md-flex flex-md-column align-items-md-center gap-md-1">
+              <div class="col-1 col-sm-2 col-md-12 d-flex justify-content-center align-content-center align-items-center"><img class="program-logo" src="./images/logos/ignite.png" alt="ignite-logo"></div>
+              <div class="col-4 col-sm-3 px-md-0 col-md-12 d-flex justify-content-start justify-content-md-center align-content-center align-items-center"><h3 class="color-red font-lato fw-bolder font-20px font-sm-25px">Network</h3></div>
+              <div class="col-7 col-md-12 px-md-0 d-flex justify-content-center align-content-center align-items-center"><p class="text-white font-1px font-sm-18px font-md-20px text-md-center">Meet professionals knowledgeable in various topics all around the world and network with them after the expo!.</p></div>
+            </div>
+          </div>  
+        </div>
+        <div id="program-join" class="d-flex justify-content-center align-items-center">
+          <a id="program-join-button" href="#" class="d-md-none text-white font-20px font-sm-25px font-lato text-decoration-none bg-red text-center">Join Expo Software 2021~2022</a>
+          <a href="#" class="d-none d-md-block text-white font-20px font-lato text-center">SEE THE HOLE PROGRAM</a>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] [OPTIONAL] This section's background image is not aligned to the original design.

Your Design       |Original Design   |
:--------------------:|:---------------------:|
![e1](https://user-images.githubusercontent.com/39523312/147301115-0b091009-b3e9-4caf-99d3-f3045480512b.png)| ![e2](https://user-images.githubusercontent.com/39523312/147301134-cadfd631-85d3-494f-baf8-bedf6506e207.png) |